base-notebook/Dockerfile

```diff
@@ -63,16 +63,16 @@ RUN mkdir /home/$NB_USER/work && \
     fix-permissions /home/$NB_USER
 
 # Install conda as jovyan and check the md5 sum provided on the download site
-ENV MINICONDA_VERSION 4.5.1
+ENV MINICONDA_VERSION 4.5.4
 RUN cd /tmp && \
     wget --quiet https://repo.continuum.io/miniconda/Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh && \
-    echo "0c28787e3126238df24c5d4858bd0744 *Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh" | md5sum -c - && \
+    echo "a946ea1d0c4a642ddf0c3a26a18bb16d *Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh" | md5sum -c - && \
     /bin/bash Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh -f -b -p $CONDA_DIR && \
     rm Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh && \
     $CONDA_DIR/bin/conda config --system --prepend channels conda-forge && \
     $CONDA_DIR/bin/conda config --system --set auto_update_conda false && \
     $CONDA_DIR/bin/conda config --system --set show_channel_urls true && \
-    $CONDA_DIR/bin/conda install --quiet --yes conda=4.5.4 && \
+    $CONDA_DIR/bin/conda install --quiet --yes conda=${MINICONDA_VERSION} && \
```

Comment:
I think this will be a no-op given the miniconda installer is using the same version, which is fine. I wonder if there's a value we should put here so we don't have to update it constantly in the long term. Maybe we can add a bit of code to lop off the patch part of `MINICONDA_VERSION` and add a * so that it updates to the latest patch? Or maybe there's a better conda-way to accomplish the same thing that @jakirkham knows?

Not a blocking issue. Just wanted to call it out for ourselves.

Reply:
Miniconda releases aren't made for every conda version, so we may well in general be installing e.g. miniconda 4.5.4 and conda 4.5.6. In general, this should not be set using the same variable, which implies the two are linked, when they aren't.